modules/audio.js

```diff
@@ -1,4 +1,5 @@
 const { Module, ProgressReport } = require('./base')
+const Resource = require('mongoose').model(resourceModel.modelName)
```

Comment:
`resourceModel` will be undefined here. Perhaps import `resourceModel` first or just replace `resourceModel.modelName` with `'resource'`